networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider.py

```diff
@@ -413,3 +425,21 @@ def sanitize(self, slice: int) -> List[Tuple[str, Callable[[str], None]]]:
         :slice: number - the number of objects that can be cleaned up at this time
         :returns: list(id, callback) - where callback is a function accepting the ID
         """
+
+
+class MigrationTracker(object):
+    def __init__(self, provider: Provider) -> None:
+        self._migration_in_progress = False
+        self.mutex = "migration-tracking"
+        self.provider = provider
+
+    def set_migration_in_progress(self, in_progress: bool):
```

Comment:
I would name it `set_migration_status`

Reply:
it is not about the status but the state, we just set a boolean flag which represents two states: "migration in progress", "migration not in progress"